### DISCUSSION TITLE
ParsePKIXPublicKey error

### DISCUSSION THREAD

**Discussion by guogeer** [golang-jwt/jwt#152] [Q&A] [answered]
https://github.com/golang-jwt/jwt/blob/4bbdd8ac624fc7a9ef7aec841c43d99b5fe65a29/ecdsa_utils.go#L54

**Comment by mfridman** [comment, marked as answer]
What error are you seeing? An example repro would be helpful..

**Reply by guogeer** [reply]
no problem!
sorry, i found no question, but i not found where can i close it
